cmd/estuary-shuttle/rpc.go

```diff
@@ -354,7 +355,8 @@ func (d *Shuttle) handleRpcAggregateContent(ctx context.Context, cmd *drpc.Aggre
 	}
 
 	if err := d.DB.Model(Pin{}).Where("id = ?", pin.ID).UpdateColumns(map[string]interface{}{
-		"active": true,
+		"active":  true,
+		"pinning": false,
```

Comment:
Can you please add a description on the statuses here on this PR?

Reply:
done